src/step-definitions/dental.steps.ts

```diff
@@ -1,16 +1,149 @@
 import { Then, When } from '@cucumber/cucumber';
 import { Context } from '../helpers/context';
 import Dental from '../page-objects/dental.page';
+import Quizzard from '../page-objects/quizzard.page';
+import Offerings from '../page-objects/offerings.page';
+import Healthcare from '../page-objects/healthcare.page';
+import { Data } from "../data/dental.data";
+import { OfferingsData } from "../data/offerings.data";
+import { HealthcareData } from "../data/healthcare.data";
+import { QuizzardData } from '../data/quizzard.data';
+import urls from "../data/urls";
+import { clickElement, getElement, getNthElement } from "../helpers/elements";
+import { expectAttributeToContain, expectText, expectToExist, expectURL, expectValue } from "../helpers/validators";
+
 
 const Page: Dental = Context.getInstance().getPage('Dental');
+const _Quizzard: Quizzard = Context.getInstance().getPage('Quizzard');
+const _Offerings: Offerings = Context.getInstance().getPage('Offerings');
+const _Healthcare: Healthcare = Context.getInstance().getPage('Healthcare');
 
 When(/^the Dental Insurance product page is opened$/, async () => {
-  await Page.validateUrl();
+  await expectURL(urls.healthcareProducts.dental, true);
+});
+
+Then(/^the Dental title and description should be correct$/, async () => {
+  await expectText(await Page.dentalTitleElem, Data.dentalTitle);
+  await expectText(await Page.dentalDescriptionElem, Data.dentalDescription);
+  await expectToExist(await Page.dentalHeaderTitleImage);
+  await expectAttributeToContain(await Page.dentalHeaderTitleImage, "xlink:href", "/brighter-assets/icons/green_icons.svg#Dental-Green");
+});
+
+Then(/^user can see Offerings link$/, async () => {
+  await expectText(await Page.dentalOfferingsLinkTextElem, Data.dentalOfferingsLinkText);
+  await expectAttributeToContain(await Page.dentalOfferingsHref, "href", "/client/indigo/offerings");
+});
+
+Then(/^user can see Healthcare link$/, async () => {
+  await expectText(await Page.dentalHealtcareLinkTextElem, Data.dentalHealtcareLinkText);
+  await expectAttributeToContain(await Page.dentalHealtcareHref, "href", "/client/indigo/offerings/606d91a9cf72104f54e0ddd9");
+});
+
+Then(/^find Offerings link is working$/, async () => {
+  await clickElement(await Page.dentalOfferingsLinkTextElem, true);
+});
+
+Then(/^the Offerings page is opened$/, async () => {
+  await expectURL(urls.offerings.base, true);
+  await expectToExist(await _Offerings.offeringsTitleElem);
+  await expectText(await _Offerings.offeringsTitleElem, OfferingsData.offeringsTitleText);
```

Comment:
Enough to expect that element exists, text of this element should be tested on Offerings test page. 
This are test for dental page, not offerings